shoes-core/lib/shoes/packager.rb

```diff
@@ -25,6 +26,17 @@ def run(path)
       @backend.run(path)
     end
 
+    def configure_gems
+      @backend.gems = []
+      return unless defined?(::Bundler)
+
+      @backend.gems = ::Bundler.environment.specs.map(&:name) - ["shoes"]
```

Comment:
FWIW, the problems with including `shoes` directly is that `warbler` still doesn't play nicely with gems that have native extensions (which ours does for the executable install).